### PR TITLE
add validation set and metrics

### PR DESCRIPTION
**Pull request by Agryosha**

### CODE REVIEW

**Review by RuslanOm — APPROVED** [review]

File: DiffAugment-stylegan2-pytorch/train.py [hunk 1]
```diff
@@ -126,6 +137,7 @@ def setup_training_loop_kwargs(
         desc += '-cond'
     else:
         args.training_set_kwargs.use_labels = False
+        args.validation_set_kwargs.use_labels = False
```
Comment (Reason239) [inline DiffAugment-stylegan2-pytorch/train.py:140]: Shouldn't validation_set_kwargs.use_labels be always True?
